Listenbrainz/Model/Feed/Metadata.swift

```diff
@@ -10,12 +10,14 @@ import Foundation
 // MARK: - Metadata
 struct Metadata: Codable {
     let trackMetadata: FeedTrackMetadata?
+    let blurbContent: String?
     let username: String?
     let userName0: String?
     let userName1: String?
 
     enum CodingKeys: String, CodingKey {
         case trackMetadata = "track_metadata"
+        case blurbContent = "blurb_content"
         case username
         case userName0 = "user_name_0"
```

Comment:
remove these

Reply:
I think I pointed to the wrong field here. I meant the username fields @gauravbhardwaj7

Reply:
@akshaaatt , I have been using these fileds in EventDescriptionView, to show the usernames in the field, imo they are necessary to show the activity which the user performs.

Reply:
Ah got it, these were to correspond to the events. I thought these were the fields that you had hardcoded a while back but that was in a different context